gulpfile.js

```diff
@@ -23,33 +21,38 @@ function errorLog (error) {
 
 // Uglify JS - Targets all .js files in the _js folder and converts
 // them to functionally identical code that uses less bytes in the _scripts folder
-gulp.task('uglify', ['lint', 'concat'], function () {
-    return gulp.src('dist/concatenated.js')
+gulp.task('uglify', ['concat'], function () {
+    return gulp.src('src/concatenated.js')
         .pipe(uglify())
         .on('error', errorLog)
         .pipe(insert.append('\n'))
-        .pipe(gulp.dest('dist'))
+        .pipe(gulp.dest('src'));
 });
 
-gulp.task('concat', ['lint'], function () {
-    return gulp.src(["node_modules/jquery/dist/jquery.js", "node_modules/marked/lib/marked.js", "src/texttransform.js", "src/!(gus-markdown*.js|*.html|*.json)"])
+gulp.task('clean', function () {
+    return gulp.src('src/concatenated.js', {read: false})
+        .pipe(clean())
+})
+
+gulp.task('concat', ['clean'], function () {
```

Comment:
Look into `concat` and see if it will auto-overwrite the existing file. If so, we don't need `clean`.